src/TTextEdit.h

```diff
@@ -93,6 +92,14 @@ class TTextEdit : public QWidget
     int mFontAscent;
     int mFontDescent;
     bool mIsCommandPopup;
+    // If this flag is set it means that this instance is currently set to
+    // display the last lines in the mpBuffer associated with the mpConsole;
+    // this setting is always true for the "lower" pane and is set/reset as
+    // needed in the upper onw. Its name seems to come from the *nix tail
+    // utility which returns the end of file(s), and, if invoked with either of
+    // the -f or -F "follow" arguments continues to show new lines as they are
+    // subsequently appended to the file with old lines scrolling up the screen
+    // which is precisely what is supposed to happen when this flag is true:
     bool mIsTailMode;
```

Comment:
8 lines to describe a member is too long - how about this:

> If true, this TTextEdit will display the last lines in `mpConsole.mpBuffer`.
> This is always `true` for the lower main window panel and it is set
> when split mode is activated for the upper one. Think of the file monitoring
> feature in tail (https://en.wikipedia.org/wiki/Tail_(Unix)#File_monitoring)

Reply:
:blush: Ermm, yeah, I do have a tendency towards calling a spade a "mechanical earth relocation device, portable, manual, for the use of"... I'll give the text a shave.

Reply:
Actually you have your logic inverted there - it is, in fact, `false` when the upper window is scrolled up from the *bottom* line (i.e. when the **`TConsole`** is *split*). :wink: